_pages/do-i-need-clearance/index.md

```diff
@@ -13,7 +13,7 @@ Not every request or collection falls under the PRA’s scope, and you may not n
 - How will you be collecting it, and
 - Why are you collecting this information
 
-This is just a starting point to see if your collection needs PRA clearance. We encourage getting in touch with your [link to contact page] agency PRA lead or OIRA Desk Officer to answer more in-depth questions.
+This is just a starting point to see if your collection needs PRA clearance. We encourage [getting in touch with your agency's PRA contact]("../contact/") to answer more in-depth questions.
```

Comment:
Because of the way Federalist works we have to do a bit of a workaround with links in order to make sure they work regardless of whether it's viewed on a local dev environment, a preview link, or the actual domain. You'll want to use an absolute link and then add this fun `relative_link` filter after it. So it'll looks something like:

`[getting in touch with your agency's PRA contact]('/contact/' | relative_url)`

Reply:
Looks great, and thanks! Go for it.